libs/sdk-ui-dashboard/.dependency-cruiser.js

```diff
@@ -31,6 +31,8 @@ options = {
                 "src/presentation/dashboardContexts",
                 "src/presentation/dialogs",
                 "src/presentation/dragAndDrop",
+                "src/presentation/dragAndDrop/types.ts",
+                "src/presentation/dragAndDrop/draggableWidget/DraggableInsightListItem.tsx",
```

Comment:
Should this be included in the `"src/presentation/dragAndDrop"` root barrel file?